### PR TITLE
fixes #11627 - address inconsistency in wording on repository details

### PR DESCRIPTION
**Pull request by bbuckingham**
Simple change... Display 'Yes' versus 'true' for consistency.

### CODE REVIEW

**Comment by bbuckingham** [comment]
[test]

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/repositories/details/views/repository-info.html [hunk 1]
```diff
@@ -127,7 +127,7 @@ <h4 translate>Basic Information</h4>
 
     <div class="detail" ng-show="repository.content_type == 'yum'">
       <span class="info-label" translate>Publish via HTTPS</span>
-      <span class="info-value" translate> true </span>
+      <span class="info-value" translate> Yes</span>
```
Comment (jlsherrill) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/repositories/details/views/repository-info.html]: Nitpick on something that was already there before, but there's an extra space before 'Yes'

**Comment by bbuckingham** [comment]
@jlsherrill, updated to remove that existing leading space

**Comment by jlsherrill** [comment]
APJ

**Comment by bbuckingham** [comment]
[test]

**Pull request merged**